DcCore/DcCore/DC/Wrapper.swift

```diff
@@ -837,12 +848,8 @@ public class DcMsg {
         return Int(dc_msg_get_from_id(messagePointer))
     }
 
-    public lazy var fromContact: DcContact = {
-        DcContact(id: fromContactId)
-    }()
-
     public var isFromCurrentSender: Bool {
-        return fromContact.id == DcContact(id: Int(DC_CONTACT_ID_SELF)).id
+        return fromContactId == DC_CONTACT_ID_SELF
```

Comment:
that will indeed save two db-calls per call to `isFromCurrentSender` - which called quite often! might be an even noticeable speed-improvement on scrolling through a chat :)